ts/src/connections/tunnelRelayTunnelHost.ts

```diff
@@ -39,17 +46,32 @@ import { SshHelpers } from './sshHelpers';
 import { MultiModeTunnelHost } from './multiModeTunnelHost';
 import { TunnelHostBase } from './tunnelHostBase';
 import { tunnelRelaySessionClass } from './tunnelRelaySessionClass';
+import { SessionPortKey } from './sessionPortKey';
+import { PortRelayConnectRequestMessage } from './messages/portRelayConnectRequestMessage';
+import { PortRelayConnectResponseMessage } from './messages/portRelayConnectResponseMessage';
 
 const webSocketSubProtocol = 'tunnel-relay-host';
```

Comment:
Maybe move these to `tunnelHostBase`? The client's ones are in `tunnelClientBase`

Reply:
The public properties are on the subclasses in both cases.

In the client case, I had to define the constants in the base source file (then import them in the subclass source file) because they are used in some implementation code in the base class.

I'm a little concerned that we might have more levels of inheritance than necessary:
`TunnelRelayTunnelClient`->`TunnelClientBase`->TunnelConnectionSession`->`TunnelConnectionBase` plus mix-ins `tunnelRelaySessionClass` and `tunnelSshSessionClass`. It has made it hard for me to find things or know where to put new code.

Some of that was supposed to allow for other connection implementations (such as LS or local network) but now I don't know if we will ever need that flexibility.

Reply:
If we don't expect to subclass any of the classes in the inheritance chain, I'm all for simplifying it. Maybe not part of this PR though.